### PR TITLE
create migration and model for Student Table

### PR DESCRIPTION
**Pull request by Giruta**

### CODE REVIEW

File: src/migrations/20200415231700-create-students-table.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    } catch (e) {
+        return Promise.reject(e);
+    }
+}
+
+/**
+ * function that sequelize-cli runs if you want to remove this migration from your database
+ * */
+export async function down(query: QueryInterface) {
+    try {
```
Comment (Alan18081) [inline src/migrations/20200415231700-create-students-table.ts:65]: We don't need try ... catch blocks here. Just return the calling of dropTable method

File: src/modules/students/students.model.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+export interface IStudent {
+    id?: number;
+    first_name_ukr: string;
+    last_name_ukr: string;
+    first_name_eng: string;
+    last_name_eng: string;
+    email: string;
+    phone_number: number;
+    password: string;
+    group_id: number;
```
[the remark this answers is not in the document]
Comment (Giruta) [reply src/modules/students/students.model.ts:12]: done

File: src/modules/students/students.model.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    createdAt?: Date;
+    updatedAt?: Date;
+}
+
+export class Student extends Model implements IStudent {
+    // public static readonly ModelName: string = 'Student';
+    // public static readonly ModelNamePlural: string = 'Students';
+    public static readonly TableName: string = 'Students';
```
[the remark this answers is not in the document]
Comment (Giruta) [reply src/modules/students/students.model.ts:20]: done

File: src/modules/students/students.model.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                    primaryKey: true,
+                    allowNull: false,
+                    autoIncrement: true,
+                },
+                first_name_ukr: new DataTypes.STRING(),
+                last_name_ukr: new DataTypes.STRING(),
+                first_name_eng: new DataTypes.STRING(),
+                last_name_eng: new DataTypes.STRING(),
+                email: new DataTypes.STRING(),
+                phone_number: new DataTypes.DOUBLE(),
```
[the remark this answers is not in the document]
Comment (Giruta) [reply src/modules/students/students.model.ts:50]: done

File: src/modules/students/students.model.ts [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+                password: new DataTypes.STRING(),
+                group_id: new DataTypes.INTEGER()
```
[the remark this answers is not in the document]
Comment (Giruta) [reply src/modules/students/students.model.ts:52]: done

File: src/migrations/20200415231700-create-students-table.ts [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            },
+            updatedAt: {
+                type: DataTypes.DATE,
+                allowNull: false,
+                comment: 'Date of the last update',
+            },
+        });
+    } catch (e) {
+        return Promise.reject(e);
+    }
```
[the remark this answers is not in the document]
Comment (Giruta) [reply src/migrations/20200415231700-create-students-table.ts:58]: deleted it

File: src/modules/students/students.model.ts [hunk 7]
```diff
@@ -0,0 +1,59 @@
+import { DataTypes, Model, ModelCtor, Sequelize } from 'sequelize';
+
+export interface IStudent {
```
[the remark this answers is not in the document]
Comment (Giruta) [reply src/modules/students/students.model.ts:3]: deleted it

File: src/migrations/20200415231700-create-students-table.ts [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    try {
+        return query.createTable(Student.TableName, {
+            id: {
+                type: DataTypes.INTEGER,
+                primaryKey: true,
+                autoIncrement: true,
+                allowNull: false,
+            },
+            first_name_ukr: {
+                type: new DataTypes.STRING(),
```
[the remark this answers is not in the document]
Comment (Giruta) [reply src/migrations/20200415231700-create-students-table.ts:14]: done

File: src/migrations/20200415231700-create-students-table.ts (hunk 1 above)
Comment (Giruta) [reply src/migrations/20200415231700-create-students-table.ts:65]: ok done

Comment (Giruta) [reply src/migrations/20200415231700-create-students-table.ts:65]: thanks

**Pull request merged**